docs/syntax/index.md

```diff
@@ -1,3 +1,6 @@
+---
+order: 50
+---
```

Comment:
This is not required.

Reply:
I only put that there because "Plugins" would appear over "Syntax", is that fine?

Reply:
Ah, right. It's okay then.
Thanks!